### PR TITLE
New profile skeleton

### PR DESCRIPTION
**Pull request by levimeahan**
Reusable character avatar component is here, also finally got around to making a `$system->isDevEnvironment()` helper method

Progress so far:
image

### CODE REVIEW

File: ui_components/src/sidebar/Sidebar.js [hunk 1]
```diff
@@ -205,7 +121,13 @@ function Sidebar({ links, navigationAPIData, userAPIData }) {
...
     return (
         <div id="sidebar">
-            {displayCharacterSection(playerData, playerResources, playerSettings, regenTime, regenOffset)}
+            <SBCharacterProfile
+                playerData={playerData}
+                playerResources={playerResources}
+                playerSettings={playerSettings}
+                regenTime={regenTime}
+                regenOffset={regenOffset}
+            />
```
Comment (levimeahan) [inline ui_components/src/sidebar/Sidebar.js:130]: TODO: do this for the rest of the sections here

File: ui_components/src/sidebar/Sidebar.js [hunk 2]
```diff
@@ -214,4 +136,111 @@ function Sidebar({ links, navigationAPIData, userAPIData }) {
...
+                            </svg>
+                        </div>
+                    </div>
+                </div>
+
+                <SBResourceBar
+                    resourceType="health"
+                    resourceAmount={playerResources.health}
+                    resourceMaxAmount={playerResources.max_health}
+                />
```
Comment (levimeahan) [inline ui_components/src/sidebar/Sidebar.js:199]: I love reusable components when they make sense

File: ui_components/src/profile/Profile.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+.stat_list.attributes {
+    width: 50%;
+}
+
+.stat {
+    width: 100%;
+    height: 84px;
+    position: relative;
+
+    font-size: 12px;
```
Comment (aumu1031) [inline ui_components/src/profile/Profile.css:60]: Can we use `var(--font-size)` variables for this? Will make it easier if we want to adjust font size between layouts in the future or allow for global layout settings (e.g. dropdown -> font size: Large).

Comment (levimeahan) [reply ui_components/src/profile/Profile.css:60]: Yup I'll update that 👍

**Pull request merged**